scripts/websocket_test.js

```diff
@@ -0,0 +1,30 @@
+let WebSocket = require('ws');
+
+let PORT_WEBSOCKET = process.env.PORT_WEBSOCKET;
+console.log('PORT_WEBSOCKET:', PORT_WEBSOCKET);
+
+let makeUrl = function () {
+	let port = PORT_WEBSOCKET || '9999';
+	return 'ws://localhost:' + port + '/ws';
+}
+
+let url = makeUrl();
+
+let langserver = new WebSocket(url);
+langserver.on('open', function open() {
+	let messages_test = [
+		'1some_rpc_message',
+		'2some_rpc_message',
+		'3some_rpc_message'
+	];
+	messages_test.map(function (message_test) {
+		console.log('sending message: %O', message_test);
+		langserver.send(message_test);
+	});
+});
+
+langserver.on('message', function (data, flags) {
+	// flags.binary will be set if a binary data is received.
+	// flags.masked will be set if the data was masked.
+});
+
```

Comment:
I'd prefer a go test so it fits it runs on CI and actually tests something. This script seems like it doesn't really assert anything.